eth/db/header.py

```diff
@@ -144,32 +146,38 @@ def header_exists(self, block_hash: Hash32) -> bool:
         validate_word(block_hash, title="Block Hash")
         return block_hash in self.db
 
-    def persist_header(self, header: BlockHeader) -> Tuple[BlockHeader, ...]:
+    def persist_header(
```

Comment:
Do you think we could refactor some of this code into a separate method that could then be reused in `ChainDB` as well?

Reply:
second to this (though lets not do this via inheritance, maybe just a helper utility function.